builder-support/specs/pdns.spec

```diff
@@ -223,7 +223,8 @@ export CPPFLAGS="-DLDAP_DEPRECATED"
   --disable-lua-records \
   --without-protobuf \
   --with-boost=/usr/include/boost148/ LDFLAGS=-L/usr/lib64/boost148 \
-  CXXFLAGS=-std=gnu++11
+  CXXFLAGS=-std=gnu++11 \
+  CPPFLAGS=-D__STDC_FORMAT_MACROS
```

Comment:
Doesn't this override the `CPPFLAGS="-DLDAP_DEPRECATED"` value set just above? Perhaps it would be better to append to `CPPFLAGS` instead of setting it?

Reply:
thanks!

Reply:
Fixing that, although apparently the build does not need it to complete :)

Reply:
amended